src/glog/logging.h.in

```diff
@@ -1016,6 +1031,41 @@ extern "C" void AnnotateBenignRaceSized(
 namespace google {
 #endif
 
+#if defined(HAVE_CXX11_CHRONO) && defined(HAVE_CXX11_ATOMIC)
+#define SOME_KIND_OF_LOG_EVERY_T(severity, seconds) \
+  static GLOG_CONSTEXPR std::chrono::duration<double, std::ratio<1, 1>> LOG_TIME_PERIOD(seconds); \
+  static std::atomic<int64_t> LOG_PREVIOUS_TIME_RAW; \
+  _GLOG_IFDEF_THREAD_SANITIZER( \
+          AnnotateBenignRaceSized(__FILE__, __LINE__, &LOG_TIME_PERIOD, sizeof(int64_t), "")); \
+  _GLOG_IFDEF_THREAD_SANITIZER( \
+          AnnotateBenignRaceSized(__FILE__, __LINE__, &LOG_PREVIOUS_TIME_RAW, sizeof(int64_t), "")); \
+  const auto LOG_CURRENT_TIME = std::chrono::steady_clock::now().time_since_epoch(); \
+  const decltype(LOG_CURRENT_TIME) LOG_PREVIOUS_TIME(LOG_PREVIOUS_TIME_RAW.load(std::memory_order_relaxed)); \
+  const auto LOG_TIME_DELTA = LOG_CURRENT_TIME - LOG_PREVIOUS_TIME; \
+  if (LOG_TIME_DELTA > LOG_TIME_PERIOD) \
+    LOG_PREVIOUS_TIME_RAW.store(LOG_CURRENT_TIME.count(), std::memory_order_relaxed); \
+  if (LOG_TIME_DELTA > LOG_TIME_PERIOD) \
+    @ac_google_namespace@::LogMessage( \
+        __FILE__, __LINE__, @ac_google_namespace@::GLOG_ ## severity).stream()
+#elif defined(OS_WINDOWS)
+#define SOME_KIND_OF_LOG_EVERY_T(severity, seconds)
+// No implementation for Windows yet
```

Comment:
What's the blocker?

Reply:
My lack of knowledge how to do this in Windows without `<chrono>`

Reply:
Do `QueryPerformanceFrequency` and `QueryPerformanceCounter` functions suffice here?